### ISSUE TITLE
guake crashing after upgrade to latest

### ISSUE THREAD

**Issue by milaad1** [Guake/guake#2108]
Hey team 
I was using guake 3.8.5 it wexcellentice and powerful
I'm using manjaro and after updating guake to 3.9 it's crushing and couldent start .. why? I reinstalled it (with paccman and add/remove) but its still doesn't work

`Guake not running, starting it
Loading Gnome schema from: /home/milad/.local/lib/python3.10/site-packages/guake/data
Unable to load the GLib schema, try to compile it
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/dbus/bus.py", line 177, in activate_name_owner
    return self.get_name_owner(bus_name)
  File "/usr/lib/python3.10/site-packages/dbus/bus.py", line 361, in get_name_owner
    return self.call_blocking(BUS_DAEMON_NAME, BUS_DAEMON_PATH,
  File "/usr/lib/python3.10/site-packages/dbus/connection.py", line 652, in call_blocking
    reply_message = self.send_message_with_reply_and_block(
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.NameHasNoOwner: Could not get owner of name 'org.guake3.RemoteControl': no such name

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/milad/.local/lib/python3.10/site-packages/guake/main.py", line 473, in main
    remote_object = bus.get_object(DBUS_NAME, DBUS_PATH)
  File "/usr/lib/python3.10/site-packages/dbus/bus.py", line 241, in get_object
    return self.ProxyObjectClass(self, bus_name, object_path,
  File "/usr/lib/python3.10/site-packages/dbus/proxies.py", line 250, in __init__
    self._named_service = conn.activate_name_owner(bus_name)
  File "/usr/lib/python3.10/site-packages/dbus/bus.py", line 182, in activate_name_owner
    self.start_service_by_name(bus_name)
  File "/usr/lib/python3.10/site-packages/dbus/bus.py", line 277, in start_service_by_name
    return (True, self.call_blocking(BUS_DAEMON_NAME, BUS_DAEMON_PATH,
  File "/usr/lib/python3.10/site-packages/dbus/connection.py", line 652, in call_blocking
    reply_message = self.send_message_with_reply_and_block(
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.ServiceUnknown: The name org.guake3.RemoteControl was not provided by any .service files

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/milad/.local/lib/python3.10/site-packages/guake/guake_app.py", line 111, in __init__
    schema_source = load_schema()
  File "/home/milad/.local/lib/python3.10/site-packages/guake/guake_app.py", line 106, in load_schema
    return Gio.SettingsSchemaSource.new_from_directory(
gi.repository.GLib.GError: g-file-error-quark: Failed to open file “/home/milad/.local/lib/python3.10/site-packages/guake/data/gschemas.compiled”: open() failed: No such file or directory (4)
Compiling schema: /home/milad/.local/lib/python3.10/site-packages/guake/data
Error opening directory “/home/milad/.local/lib/python3.10/site-packages/guake/data”: No such file or directory
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/dbus/bus.py", line 177, in activate_name_owner
    return self.get_name_owner(bus_name)
  File "/usr/lib/python3.10/site-packages/dbus/bus.py", line 361, in get_name_owner
    return self.call_blocking(BUS_DAEMON_NAME, BUS_DAEMON_PATH,
  File "/usr/lib/python3.10/site-packages/dbus/connection.py", line 652, in call_blocking
    reply_message = self.send_message_with_reply_and_block(
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.NameHasNoOwner: Could not get owner of name 'org.guake3.RemoteControl': no such name

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/milad/.local/lib/python3.10/site-packages/guake/main.py", line 473, in main
    remote_object = bus.get_object(DBUS_NAME, DBUS_PATH)
  File "/usr/lib/python3.10/site-packages/dbus/bus.py", line 241, in get_object
    return self.ProxyObjectClass(self, bus_name, object_path,
  File "/usr/lib/python3.10/site-packages/dbus/proxies.py", line 250, in __init__
    self._named_service = conn.activate_name_owner(bus_name)
  File "/usr/lib/python3.10/site-packages/dbus/bus.py", line 182, in activate_name_owner
    self.start_service_by_name(bus_name)
  File "/usr/lib/python3.10/site-packages/dbus/bus.py", line 277, in start_service_by_name
    return (True, self.call_blocking(BUS_DAEMON_NAME, BUS_DAEMON_PATH,
  File "/usr/lib/python3.10/site-packages/dbus/connection.py", line 652, in call_blocking
    reply_message = self.send_message_with_reply_and_block(
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.ServiceUnknown: The name org.guake3.RemoteControl was not provided by any .service files

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/milad/.local/lib/python3.10/site-packages/guake/guake_app.py", line 111, in __init__
    schema_source = load_schema()
  File "/home/milad/.local/lib/python3.10/site-packages/guake/guake_app.py", line 106, in load_schema
    return Gio.SettingsSchemaSource.new_from_directory(
gi.repository.GLib.GError: g-file-error-quark: Failed to open file “/home/milad/.local/lib/python3.10/site-packages/guake/data/gschemas.compiled”: open() failed: No such file or directory (4)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/milad/.local/bin/guake", line 8, in <module>
    sys.exit(exec_main())
  File "/home/milad/.local/lib/python3.10/site-packages/guake/main.py", line 648, in exec_main
    if not main():
  File "/home/milad/.local/lib/python3.10/site-packages/guake/main.py", line 489, in main
    instance = Guake()
  File "/home/milad/.local/lib/python3.10/site-packages/guake/guake_app.py", line 114, in __init__
    try_to_compile_glib_schemas()
  File "/home/milad/.local/lib/python3.10/site-packages/guake/paths.py", line 96, in try_to_compile_glib_schemas
    subprocess.check_call(["glib-compile-schemas", "--strict", SCHEMA_DIR])
  File "/usr/lib/python3.10/subprocess.py", line 369, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['glib-compile-schemas', '--strict', '/home/milad/.local/lib/python3.10/site-packages/guake/data']' returned non-zero exit status 1.
`

-------------------------------------------------------------------------------

`╰─λ guake --support                                                                                                                                                                 1 (0.290s)
<details><summary>$ guake --support</summary>

Guake Version:		3.9.0

Vte Version:		0.68.0

Vte Runtime Version:	0.68.0

--------------------------------------------------
GTK+ Version:		3.24.34

GDK Backend:		<GdkX11.X11Display

--------------------------------------------------
Desktop Session: xfce

--------------------------------------------------
Display: :0.0

RGBA visual: True

Composited: True

* Monitor: 0 - GSM HDMI-2
    * Geometry:		1920 x 1080 at 0, 0
    * Size:		480 x 270 mm²
    * Primary:		True
    * Refresh rate:	60.0 Hz
    * Subpixel layout:	unknown
* Monitor: 1 - BOE eDP-1
    * Geometry:		1920 x 1080 at 0, 1080
    * Size:		309 x 174 mm²
    * Primary:		False
    * Refresh rate:	60.0 Hz
    * Subpixel layout:	unknown
`

**Comment by milaad1** [comment]
Here I have a output from running guake with sudo ! it's freezing in the last line
`[sudo] password for milad: 
Guake not running, starting it
Loading Gnome schema from: /usr/share/glib-2.0/schemas
Language previously loaded from: /usr/share/locale
Guake Terminal 3.9.1.dev0
VTE 0.68.0
Gtk 3.24.34
created fresh notebook for workspace 0
Spawning new terminal at /root
current workspace is 0
can't bind show-focus key
Cannot find session.json file
Guake initialized
`
and yes I delete session.json for clean reinstall :/

**Comment by tejaskale** [comment]
Installed guake 3.9 from pip today , and ran into the same issue .

But looking at the error message, it seems that pip install doesn't have the  the  `/home/<username>/.local/lib/python3.9/site-packages/guake/data` directory . 

Simply copying this form the repository to that location made this work. 
So workaround for now : 

- Clone this repo
- Install guake with pip
- Copy the data directory  to `/home/<username>/.local/lib/python3.9/site-packages/guake/data` 
    - or whatever location the error message above reports
 - Try to start guake normally. This should compile the glib schema automatically.

**Comment by milaad1** [comment]
Hey thanks for your reply..
i did this and i have something new !
```

guake                                                                                                                                                                           0 (8.387s)
Guake not running, starting it
Loading Gnome schema from: /usr/share/glib-2.0/schemas
Language previously loaded from: /usr/share/locale
Guake Terminal 3.9.1.dev0
VTE 0.68.0
Gtk 3.24.34
created fresh notebook for workspace 0
Spawning new terminal at /home/milad
current workspace is 0
can't bind show-focus key
Cannot find session.json file
Guake initialized
Calling startup script: You Again ?
Traceback (most recent call last):
  File "/home/milad/.local/bin/guake", line 8, in <module>
    sys.exit(exec_main())
  File "/usr/lib/python3.10/site-packages/guake/main.py", line 648, in exec_main
    if not main():
  File "/usr/lib/python3.10/site-packages/guake/main.py", line 630, in main
    pid = subprocess.Popen(  # pylint: disable=consider-using-with
  File "/usr/lib/python3.10/subprocess.py", line 969, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.10/subprocess.py", line 1845, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'You Again ?'

```

**Comment by milaad1** [comment]
Ok let me end this issue
I just run guake with -u for disabling startup script --no-startup-script..
after guake run, I open the properties and remove hooks 'You Again ?'. Now it works nice and easy.

**Comment by carloshimura** [comment]
I am facing the same issue (arch linux)
The mentioned solution installing from PIP didn't work for me.


>> uname -r
5.15.54-1-lts

>> guake


Guake not running, starting it
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/dbus/bus.py", line 177, in activate_name_owner
    return self.get_name_owner(bus_name)
  File "/usr/lib/python3.10/site-packages/dbus/bus.py", line 361, in get_name_owner
    return self.call_blocking(BUS_DAEMON_NAME, BUS_DAEMON_PATH,
  File "/usr/lib/python3.10/site-packages/dbus/connection.py", line 652, in call_blocking
    reply_message = self.send_message_with_reply_and_block(
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.NameHasNoOwner: Could not get owner of name 'org.guake3.RemoteControl': no such name





>> guake --support
<details><summary>$ guake --support</summary>

Guake Version:		3.9.1.dev0

Vte Version:		0.68.0

Vte Runtime Version:	0.68.0

--------------------------------------------------
GTK+ Version:		3.24.34

GDK Backend:		<GdkX11.X11Display

--------------------------------------------------
Desktop Session: gnome

--------------------------------------------------
Display: :1

RGBA visual: True

Composited: True

* Monitor: 0 - SDC DP-2
    * Geometry:		2560 x 1440 at 0, 0
    * Size:		344 x 194 mm²
    * Primary:		True
    * Refresh rate:	60.0 Hz
    * Subpixel layout:	unknown
* Monitor: 1 - AOC HDMI-0
    * Geometry:		2560 x 1440 at 2560, 0
    * Size:		527 x 296 mm²
    * Primary:		False
    * Refresh rate:	59.95 Hz
    * Subpixel layout:	unknown

**Comment by Isegrimm666** [comment]
Hi ...

I have still the same issue ... and nothing of the posted ways above work for me.

System: Debian siduction, kernel 5.18-12-1-siduction-amd64
Desktop: cinnamon

Starting Guake in a Terminal generates the following Error:

`isegrimm@THANATOS:~$ guake
Guake not running, starting it
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/dbus/bus.py", line 177, in activate_name_owner
    return self.get_name_owner(bus_name)
  File "/usr/lib/python3/dist-packages/dbus/bus.py", line 361, in get_name_owner
    return self.call_blocking(BUS_DAEMON_NAME, BUS_DAEMON_PATH,
  File "/usr/lib/python3/dist-packages/dbus/connection.py", line 652, in call_blocking
    reply_message = self.send_message_with_reply_and_block(
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.NameHasNoOwner: Could not get owner of name 'org.guake3.RemoteControl': no such name

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/guake/main.py", line 473, in main
    remote_object = bus.get_object(DBUS_NAME, DBUS_PATH)
  File "/usr/lib/python3/dist-packages/dbus/bus.py", line 241, in get_object
    return self.ProxyObjectClass(self, bus_name, object_path,
  File "/usr/lib/python3/dist-packages/dbus/proxies.py", line 250, in __init__
    self._named_service = conn.activate_name_owner(bus_name)
  File "/usr/lib/python3/dist-packages/dbus/bus.py", line 182, in activate_name_owner
    self.start_service_by_name(bus_name)
  File "/usr/lib/python3/dist-packages/dbus/bus.py", line 277, in start_service_by_name
    return (True, self.call_blocking(BUS_DAEMON_NAME, BUS_DAEMON_PATH,
  File "/usr/lib/python3/dist-packages/dbus/connection.py", line 652, in call_blocking
    reply_message = self.send_message_with_reply_and_block(
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.ServiceUnknown: The name org.guake3.RemoteControl was not provided by any .service files

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/guake", line 33, in <module>
    sys.exit(load_entry_point('guake==3.9.0', 'console_scripts', 'guake')())
  File "/usr/lib/python3/dist-packages/guake/main.py", line 648, in exec_main
    if not main():
  File "/usr/lib/python3/dist-packages/guake/main.py", line 487, in main
    from guake.guake_app import Guake
  File "/usr/lib/python3/dist-packages/guake/guake_app.py", line 49, in <module>
    from guake import notifier
  File "/usr/lib/python3/dist-packages/guake/notifier.py", line 23, in <module>
    gi.require_version("Notify", "0.7")
  File "/usr/lib/python3/dist-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available for version %s' %
ValueError: Namespace Notify not available for version 0.7
`
Following some hints in the web, i deinstalled guake, removed the folder "guake" from "/usr/lib/python3/dist-packages" ... and reinstalled guake using apt install guake.

Rebooting ... and the same issue appeared.

Any suggestions?

Regards
Stefan
